### PR TITLE
creating models and migrations

### PR DESCRIPTION
**Pull request by oramasangel**
# Description
What did you implemented/Why did you implemented this:
 - i've create the models for the database migrations
 
 Examples:

### CODE REVIEW

File: local_sales/test/fixtures/categories.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+
+two:
+  name: MyString
```
Comment (juliocabrera820) [inline local_sales/test/fixtures/categories.yml:7]: this must be removed

File: local_sales/test/fixtures/companies.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+  adress: MyString
+
+two:
+  name: MyString
+  adress: MyString
```
Comment (juliocabrera820) [inline local_sales/test/fixtures/companies.yml:9]: this must be removed

File: local_sales/test/fixtures/offers.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+one:
+  name: MyString
+  status: 1
+  discount: 1.5
+
+two:
+  name: MyString
+  status: 1
+  discount: 1.5
```
Comment (juliocabrera820) [inline local_sales/test/fixtures/offers.yml:11]: this must be removed

File: local_sales/test/fixtures/products.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  name: MyString
+  price: 1.5
+  quantity: 1
+  description: MyString
+
+two:
+  name: MyString
+  price: 1.5
+  quantity: 1
+  description: MyString
```
Comment (juliocabrera820) [inline local_sales/test/fixtures/products.yml:13]: this must be removed

File: local_sales/test/fixtures/providers.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+
+two:
+  name: MyString
```
Comment (juliocabrera820) [inline local_sales/test/fixtures/providers.yml:7]: this must be removed

File: local_sales/test/models/category_test.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class CategoryTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (juliocabrera820) [inline local_sales/test/models/category_test.rb:7]: this must be removed

File: local_sales/test/models/company_test.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class CompanyTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (juliocabrera820) [inline local_sales/test/models/company_test.rb:7]: this must be removed

File: local_sales/test/models/offer_test.rb [hunk 8]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class OfferTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (juliocabrera820) [inline local_sales/test/models/offer_test.rb:7]: this must be removed

File: local_sales/test/models/product_test.rb [hunk 9]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class ProductTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (juliocabrera820) [inline local_sales/test/models/product_test.rb:7]: this must be removed

File: local_sales/test/models/provider_test.rb [hunk 10]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class ProviderTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (juliocabrera820) [inline local_sales/test/models/provider_test.rb:7]: this must be removed

**Review by brayandiazc — APPROVED** [review]
ok

**Review by juliocabrera820 — APPROVED** [review]
Nice job 👍🏼

**Review by JoseRicardoJimenez — APPROVED** [review]
fine

**Pull request merged**